kinesis-video-pic/src/utils/src/Mutex.c

```diff
@@ -170,29 +170,17 @@ STATUS defaultConditionVariableWait(CVAR cvar, MUTEX mutex, UINT64 timeout)
 
 #else
 
-// Definition of the static global mutexes
```

Comment:
Why are you removing these?

Reply:
We should not shared a  global mutexes  if mutex creation was failed. It should assert instead.

Reply:
Agree. We are not using these anywhere and none of the customers (in my opinion) need these.